pkg/detectors/captaindata/v3/captaindata.go

```diff
@@ -0,0 +1,110 @@
+package captaindata
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
```

Comment:
The versioning interface is currently unimplemented. When running trufflehog, the following warning is displayed:
```
2024-10-25T11:39:18+05:00	info-0	trufflehog	possible duplicate detector configured	{"detector": "CaptainData"}
```

Reply:
How are you testing this @kashifkhan0771 , can you tell me the command?

Reply:
Add your token in some file locally and run following command against it
```
trufflehog filesystem ~/<path_to_file_where_secret_is>
```

Reply:
Thanks, I checked with the changes, this warning is not coming now, I have fixed it.